### ISSUE TITLE
Cannot adjust Kanban Board Helper

### ISSUE THREAD

**Issue by hexagonsun24** [chhoumann/MetaEdit#35]
Second time this has happened, I create a new Kanban board, and try to link it to a YAML Metaedit field, but when I try to click on the gear icon beside the Kanban Board Helper, nothing happens. At this stage, Kanban linked metadata changes no longer work either. Tried closing app, toggling enable/disable, restarting computer. Only seems to be solved by uninstalling entirely.

**Comment by liammrharris** [comment]
+1 Same issue here.

I have a few kanban boards set up with metaedit’s helper.

1. I copied the plugin file to another location.
2. I uninstalled metaedit
3. I reinstalled it from the Community Plugins browser in Obsidian settings.
4. I went to metaedit’s settings and found that the kanban helper’s settings are now accessible upon clicking the relevant gear icon.
5. I copied over the data.json file into the current installation from the old installation I had copied to another directory (attempting to lazily get back my settings.
6. Same issue returned. Can’t use the kanban helper’s gear icon.
